### DISCUSSION TITLE
Add pods to react native module

### DISCUSSION THREAD

**Discussion by Garciax** [callstack/react-native-builder-bob#406] [Q&A]
## Add pod  in react native module
I'm trying to add pods to my module, but no dependency is found when building the project.
How to correctly add pods to your module?

<img width="617" alt="Снимок экрана 2020-12-19 в 14 06 50" src="https://user-images.githubusercontent.com/32370985/102687959-5d723400-4204-11eb-81e7-17ba98245bee.png">
<img width="742" alt="Снимок экрана 2020-12-19 в 14 11 46" src="https://user-images.githubusercontent.com/32370985/102687964-6400ab80-4204-11eb-97b0-e0c6220020eb.png">

**Comment by jonathanmachado** [comment]
same here

**Comment by crherman7** [comment]
@Garciax why are you not using `<project>.podspec` and adding the dependency there?

```
s.dependency "Alamofire", "~> 5.4.1"
```

**Comment by a-eid** [comment]
as @crherman7 said, you only need to add the `dependency` inside the `podspec`.
when the app the uses the packages issues `pod install` they dependency will get installed.

**Comment by Lg0gs** [comment]
@Garciax 
Any updates?
